spec/dom-to-image-more.spec.js

```diff
@@ -174,8 +171,19 @@
                     .catch(done);
             });
 
+            it('should render bare text nodes not wrapped in an element', function (done) {
+                this.timeout(30000);
+                loadTestPage('bare-text-nodes/dom-node.html', 'bare-text-nodes/style.css')
+                    // NOTE: Using first child node of domNode()!
+                    .then((node) => renderChildToPng(node)) //, { width: 200, height: 200 }))
```

Comment:
The interesting bit here is that we're using a different helper because we have to get the `domNode()` first child to get a bare text node.

Also note that without intrinsic sizing fix, we could pass a specific width and height using the support that was always there in `options`